adr/0001-community-discussions.adoc

```diff
@@ -0,0 +1,70 @@
+
+Contributors want to arrange or attend an event around Quarkus - where do they post/look for info on that ?
+
+Contributor/committer are trying to follow new questions and issues - what does he need to track to do that ? today it is alot and it is all different notification sytems and mainly its a big "firehose" with no good ability to filter out what you don't need.
+
+User looking for a job involving Quarkus - where do they go look ?
+
+
+////
+ as well as a lot of social platform presence https://twitter.com/quarkusio/[twitter], https://www.facebook.com/quarkusio/[facebook], https://www.linkedin.com/groups/13789086/[linkedin], https://www.youtube.com/quarkusio[youtube] and https://www.linkedin.com/groups/13789086/[reddit].
+////
+
+== Considered options
+
+* stackoverflow will not go away - thus for this current proposal it does not include Q/A section.
+* "I can't track more places" is a very valid worry; the advantage of discussions is that it shows up in your normal GitHub notifications which you can mute, filter,etc. like anything else on GitHub. Thus it gives you more control rather than just adding more noise.
+
+=== quarkusio-community mailing list
+Not proposed since it is just another waterhose of info with no good ability to ignore/consume as needed/wanted.
+
+=== enable discussions on github.com/quarkusio/quarkus-insights
+Considered as quarkus insights is an existing "community" and let us have separate permissions for moderators/triage but some suggested it might be hard to find.
+
+=== enable discussions on github.com/quarkusio/community
+Downside is that labels etc. would need to be kept in sync and that searches in github.com/quarkusio would not show up community/discussions.
+It could show up if searched on github.com/quarkus level though.
+Could be considered as allow us to have separate permissions for moderator/triagers separate from commit access - need to check if https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization[github docs] on permissions levels work for us.
+
+== Decision
+
+Enable GitHub discussions feature on `quarkusio/quarkus` with the following initial categories:
+
+- Announcements (post only by admins)
+- Quarkus Community (post by anyone, general discussions)
+- Quarkus Insights Episodes (show notes and offline comments)
+- Quarkus World Tour and other Events (Setup and announce of interest or other events)
+- Jobs and Opportunities (post by anyone)
+
+== Consequences
+
+With this the conversation(s) in Quarkus community is no longer only about development on quarkus-dev mailing list nor  just by Quarkus core contributors. That should enable broader participation and let Quarkus community grow beyond just "the code".
+
+We don't (yet) add a Q/A section to this as Zulip Users and Stackoverflow still exist.
```

Comment:
The first sections talks a lot about user Q&A, the fact they sometime don't feel compfortable to post on Zulip and the fac that Zulip is not searchable so I don't understand this decision.
Maybe a "Seek for help" section would be needed.

Reply:
was missing a commit - updated.